From 271f5c8380c7ea2d230419baf999525b1f2ff4d9 Mon Sep 17 00:00:00 2001
From: Jaikrishna Nemallapudi <nemallapudi.jaikrishna@intel.com>
Date: Wed, 3 Mar 2021 15:48:37 +0530
Subject: [PATCH 2/2] Changes to fix compatibility issues with ngraph ops

The eltwise add op for bias causes crash due to parallel
APIs called in mkldnn_eltwise_node.cpp.
Hence, enabled legacy _WIN32 for this file to prevent it.

Tracked-On: OAM-96219
Signed-off-by: Jaikrishna Nemallapudi <nemallapudi.jaikrishna@intel.com>
Signed-off-by: akodanka <anoob.anto.kodankandath@intel.com>
Signed-off-by: Jeevaka Prabu Badrappan <jeevaka.badrappan@intel.com>
---
 inference-engine/src/mkldnn_plugin/nodes/mkldnn_eltwise_node.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/inference-engine/src/mkldnn_plugin/nodes/mkldnn_eltwise_node.cpp b/inference-engine/src/mkldnn_plugin/nodes/mkldnn_eltwise_node.cpp
index 856bac838..e7c52b6a9 100644
--- a/inference-engine/src/mkldnn_plugin/nodes/mkldnn_eltwise_node.cpp
+++ b/inference-engine/src/mkldnn_plugin/nodes/mkldnn_eltwise_node.cpp
@@ -24,6 +24,7 @@
 #include "mkldnn_pooling_node.h"
 #include <mkldnn_selective_build.h>
 
+#define _WIN32 1
 using namespace MKLDNNPlugin;
 using namespace InferenceEngine;
 using namespace mkldnn::impl::utils;
-- 
2.17.1

